contributing/community.ipynb

```diff
@@ -0,0 +1,68 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9cbbef13-a05c-4169-9627-e54a43a11ee1",
+   "metadata": {},
+   "source": [
+    "# Research Coding Community (RCC)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "629d3135-1e5a-41dd-bae3-638d4af4e899",
+   "metadata": {},
+   "source": [
+    "The Research Coding Community (RCC) is a network brought together for everyone across the university who writes code in support of their research. Having just completed your CfRR course, signing up to the RCC is the next step in connecting with peers who share your interests and challenges in computational research.\n",
```

Comment:
Could we amend this slightly, ensuring members of Professional Services feel included?
How about the following?

> The Research Coding Community (RCC) is a network to bring together everyone across the university who writes code in support of research. Having just completed your CfRR course, signing up to the RCC is the next step in connecting with peers who share your interests and challenges.